examples/llm_and_nlp/unstructured-embeddings-gen.py

```diff
@@ -0,0 +1,69 @@
+from collections.abc import Iterator
```

Comment:
[C] In other examples there is usually a list of packages to install that don't come with the base datachain install.

Reply:
The idea is to have the same kind of block as shown in https://github.com/iterative/datachain/pull/426 unless a user installs `datachain[examples]` this example won't work.